source/adapters/level_zero/context.cpp

```diff
@@ -36,7 +38,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urContextCreate(
   ZE2UR_CALL(zeContextCreate, (Platform->ZeDriver, &ContextDesc, &ZeContext));
   try {
     ur_context_handle_t_ *Context =
-        new ur_context_handle_t_(ZeContext, DeviceCount, Devices, true);
+        new v2::ur_context_handle_t_(ZeContext, DeviceCount, Devices, true);
```

Comment:
This still defaults to non v2 somehow?

Reply:
See the relevant [commit message](https://github.com/oneapi-src/unified-runtime/pull/1801/commits/58ddeb2aa8098adb06dcf6c653476f3cd38afb59):
```
that inherits from the old context and extends it with v2 functionality.
urContextCreate will now always create v2::context. All legacy
functions will continue to operate as normal and the optimized
code path can levarage additional features implemented in v2.
```

Reply:
Cool, thanks.